docs/manual/src/Overview.md

```diff
@@ -16,4 +23,4 @@ This .udl (Uniffi Definition Language) file, whose definitions must match with t
 - Kotlin
 - Swift
 - Python
-- [Gecko](https://en.wikipedia.org/wiki/Gecko_(software)) C++
+- [Gecko](https://en.wikipedia.org/wiki/Gecko_(software)) JavaScript
```

Comment:
clarifying question: Is this a necessary part of this change or tangentially related?

Reply:
completely unnecessary tangent